mesh/mesh.hpp

```diff
@@ -1960,6 +1962,17 @@ class Mesh
                           Array<IntegrationPoint>& ips, bool warn = true,
                           InverseElementTransformation *inv_trans = NULL);
 
+   /** @brief Computes geometric parameters associated with a Jacobian matrix
+       in 2D/3D. These parameters are
+       (i) Area/Volume,
+       (ii) Aspect-ratio (1 in 2D, and 2 dimensional and 2 non-dimensional
+                          parameters in 3D. Dimensional parameters are used
+                          for target construction in TMOP),
+       (iii) skewness (1 in 2D and 3 in 3D), and finally
+       (iv) orientation (1 in 2D and 3 in 3D).
+    */
+   void GetGeometricParametersFromJacobian(const DenseMatrix &J, Vector &par);
```

Comment:
Does it all have to be in one `Vector`? It would be simpler if the output is 1 double and 3 vectors.